lib/matplotlib/tests/test_afm.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.afm as afm
```

Comment:
In the future, if anyone adds new tests here, then they won't have to worry about adding in these imports if necessary. Easy consistency is important for tests.